tests/DirectionProvider_test.jsx

```diff
@@ -3,47 +3,47 @@ import { expect } from 'chai';
 import { shallow } from 'enzyme';
 import sinon from 'sinon-sandbox';
 
+import { CHANNEL, DIRECTIONS, defaultDirection } from '../src/constants';
 import DirectionProvider from '../src/DirectionProvider';
-import { DIRECTIONS } from '../src/constants';
+import mockBrcast from './mocks/brcast_mock';
 
-describe('<DirectionProvider>', () => {
-  let children;
-  beforeEach(() => {
-    children = <div>Foo</div>;
-  });
+function getWrapper(props, context) {
```

Comment:
It's very very preferred to repeat the jsx in every test, and *not* to create "getWrapper" functions. In addition, this creates a lot of diff churn, making this PR harder to review. Could we revert this change?

Reply:
My bad! And noted for the future.